### DISCUSSION TITLE
Additional Alert Support

### DISCUSSION THREAD

**Discussion by hobybrenner** [moghtech/komodo#225] [Ideas]
Is there any chance at adding support Apprise for alerting?  This would open up alerting for many services.

Apprise allows you to send a notification to almost all of the most popular notification services available to us today such as: Telegram, Discord, Slack, Amazon SNS, Gotify, etc.

https://github.com/caronc/apprise

Thanks!

**Comment by apat183** [comment]
Would be great to add something that can expand, there are other services like this that can be implemented that open up lots of alerting.
